File: src/Microsoft.AspNetCore.SignalR.Common/HandshakeRequestMessage.cs
// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.

namespace Microsoft.AspNetCore.SignalR.Protocol
{
    public class HandshakeRequestMessage : HubMessage
    {
        public HandshakeRequestMessage(string protocol, int version)
        {
            Protocol = protocol;
            Version = version;
        }

        public string Protocol { get; }
        public int Version { get; }
    }
}

File: src/SignalR/clients/java/signalr/src/main/java/com/microsoft/signalr/HandshakeRequestMessage.java
// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.

package com.microsoft.signalr;

final class HandshakeRequestMessage {
    private final String protocol;
    private final int version;

    public HandshakeRequestMessage(String protocol, int version) {
        this.protocol = protocol;
        this.version = version;
    }
}
